cisco_node_utils.gemspec

```diff
@@ -31,6 +31,7 @@ Currently supports NX-OS nodes.
   spec.add_development_dependency 'bundler', '~> 1.7'
   spec.add_development_dependency 'kwalify', '~> 0.7.2'
   spec.add_development_dependency 'minitest', '~> 5.0'
+  spec.add_development_dependency 'net-telnet', '~> 0.1.1'
```

Comment:
Why is this needed?

Reply:
It's required by basetest

https://github.com/cisco/cisco-network-node-utils/blob/develop/tests/basetest.rb#L32

Reply:
Good catch.  Thanks for adding that in.